comparisons/comparison-brigade.md

```diff
@@ -403,18 +403,18 @@ states:
   - eventRefs:
     - execEvent
     eventDataFilter:
-      dataOutputPath: "{{ $.event }}"
+      dataOutputPath: "${ .event }"
     actions:
     - name: eventInfoAction
       functionRef:
         refName: consoleFunction
         parameters:
-          log: ">>> event $event.type caused by $.event.data.provider"
+          log: ">>> event ${ .event.type } caused by ${ .event.data.provider }"
```

Comment:
String formatting seems like a third kind of use for expression languages. It looks familiar, is there some common string formatting in e.g. helm templating that can be used?

Reply:
sorry idk much about helm. could you show example so i understand what this means?